packages/instantsearch-ui-components/src/components/Hits.tsx

```diff
@@ -40,31 +93,38 @@ export type HitsClassNames = {
    * Class names to apply to each item element
    */
   item: string | string[];
+  /**
+   * Class names to apply to the banner element
+   */
+  bannerRoot: string | string[];
+  /**
+   * Class names to apply to the banner image element
+   */
+  bannerImage: string | string[];
+  /**
+   * Class names to apply to the banner link element
+   */
+  bannerLink: string | string[];
 };
 
-export function createHitsComponent({ createElement }: Renderer) {
+export function createHitsComponent({ createElement, Fragment }: Renderer) {
```

Comment:
Probably because `Fragment` is not marked as optional when it should be as Vue 2 doesn't support it.
Guess it's not too big of a deal though, as Vue InstantSearch is written in JS so this won't error 😆